lib/devices/android/android.js

```diff
@@ -557,6 +558,18 @@ Android.prototype.waitForActivityToStop = function (cb) {
   this.adb.waitForNotActivity(this.args.appWaitPackage, this.args.appWaitActivity, cb);
 };
 
+Android.prototype.setActualCapabilities = function (cb) {
+  this.capabilities.deviceName = this.adb.udid || this.adb.curDeviceId;
+  this.adb.shell("getprop ro.build.version.release", function (err, version) {
+    if (err) {
+      logger.warn(err);
+    } else {
+      logger.debug("Device is at release version " + version);
+      this.capabilities.platformVersion = version;
+    }
+  }.bind(this));
+  return cb();
```

Comment:
last comment: if we don't need to short-circuit the logic, our style is to not use `return` here.

we can simply do `cb();`